src/app/modules/item/pages/item-edit-advanced-parameters/item-edit-advanced-parameters.component.ts

```diff
@@ -1,8 +1,15 @@
 import { Component, Input } from '@angular/core';
-import { FormGroup } from '@angular/forms';
+import { FormGroup, Validators } from '@angular/forms';
 import { DropdownOption } from 'src/app/modules/shared-components/components/dropdown/dropdown.component';
 import { Item } from '../../http-services/get-item-by-id.service';
 
+const MIN_MAX_ALLOW_RANGE_HOURS = 1;
+const MIN_MAX_ALLOW_RANGE_MS = MIN_MAX_ALLOW_RANGE_HOURS * 3600 * 1000;
+
+function getAllowedTimeMaxDate(timeMinDate: Date): Date {
```

Comment:
Fixed

Reply:
> Anyway, it is only used at one place, it would be probably more readable to have this directly into the code.

As it is used once, it is is better to use the operation directly. Where it is called, even the new name is not clear.

Reply:
Fixed